Examples/Physics_applications/free_electron_laser/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _examples-free-electron-laser:
+
+Free-electron laser
+===================
+
+This example shows how to simulate the physics of a free-electron laser (FEL) using WarpX.
+In this example, a relativistic electron beam is sent through an undulator (represented by an external,
+oscillating magnetic field). The radiation emitted by the beam grows exponentially
+as the beam travels through the undulator, due to the Free-Electron-Laser instability.
+
+The parameters of the simulation are taken from section 5.1 of :cite:t:`ex-Fallahi2020`.
+
+The simulation is performed in 1D, and uses the boosted-frame technique as described in
+:cite:t:`ex-VayFELA2009` and :cite:t:`ex-VayFELB2009`, to reduce the computational cost of the simulation.
+(The Lorentz frame of the simulation is moving at the average speed of the beam in the undulator.)
```

Comment:
Could we make the parenthesis part of the sentence that it refers to here (if I understood correctly)?
```suggestion
The simulation is performed in 1D, and uses the boosted-frame technique as described in
:cite:t:`ex-VayFELA2009` and :cite:t:`ex-VayFELB2009` to reduce the computational cost (the Lorentz frame of the simulation is moving at the average speed of the beam in the undulator).
```